spec/mailers/prior_authority/submission_feedback_mailer_spec.rb

```diff
@@ -133,7 +133,7 @@
   end
 
   context 'with other state for further information request' do
-    let(:state) { 'fake' }
+    let(:state) { 'sent_back' }
```

Comment:
i think this file needs rebasing as it is not showing latest intent for this test. or the one above at line 119

Reply:
it should be just testing an invalid state. 

```
context 'with other state' do
```

Reply:
@jsugarman the issue is we now have validation that prevents unrecognised states being persisted. But maybe that's a sign this spec is now redundant.

Reply:
yes, perhaps, although the latest version is just testing for error capture and raising of invalid states. The test at 119 above it, in main, is already exercising the 'sent_back' state.

Reply:
Ok, rebased now. It's impossible for a PA application to be assigned a state that's not in its list, but the spec still works if we assign a state that's valid but we don't have a feedback mailer for. Like `submitted`.